broker/api.py

```diff
@@ -76,10 +76,10 @@ def catalog(self) -> Service:
                     description="Custom domain with TLS and CloudFront.",
                 ),
                 ServicePlan(
-                    id=MIGRATION_PLAN_ID, 
+                    id=MIGRATION_PLAN_ID,
                     name="migration-not-for-direct-use",
                     description="Migration plan for internal autmation.",
-                )
+                ),
```

Comment:
These are just from running `black` - I forgot to run it last time I pushed 😬

Reply:
Should we add a lint step to CI?

Reply:
🤷 I'm not that concerned about making sure stuff stays 100% linted - I just use `black` periodically so it doesn't get out of control. I'd hate to have a necessary change have to go through a second round of PRs because of an extra space
since `black` is a formatter, we can't add it to CI unless/until we figure out a way to let bots commit

Reply:
Ah, just realized we don't have CI setup on PR's. Does caulking not like other pre-commit hooks on repos?

Reply:
It does not: cloud-gov/caulking#37